src/components/pages/Home/HomeContainer.js

```diff
@@ -33,8 +42,17 @@ function HomeContainer({ LoadingComponent }) {
       {authState.isAuthenticated && !userInfo && (
         <LoadingComponent message="Fetching user profile..." />
       )}
-      {authState.isAuthenticated && userInfo && (
+
+      {authState.isAuthenticated &&
+      userInfo &&
+      oktaIds &&
+      !(userInfo.oktaId in oktaIds) ? (
         <RenderHomePage userInfo={userInfo} authService={authService} />
+      ) : (
+        // you can either render a registration component here,
+        <h1>you need to register before you can view dashboard</h1>
+        // OR you can redirect to it's own registration page
+        // <Redirect to={'/registration'} />
```

Comment:
Right here is where you'd either render a registration component, or redirect to a new page

Reply:
Some very nice work Cory!